SDWebImage/SDImageGraphics.m

```diff
@@ -46,7 +46,7 @@ CGContextRef SDGraphicsGetCurrentContext(void) {
 
 void SDGraphicsBeginImageContext(CGSize size) {
 #if SD_UIKIT || SD_WATCH
-    SDGraphicsBeginImageContext(size);
+    UIGraphicsBeginImageContext(size);
 #else
     SDGraphicsBeginImageContextWithOptions(size, NO, 1.0);
 #endif
```

Comment:
Small suggests, remove these code,`SDGraphicsBeginImageContext` just call `SDGraphicsBeginImageContextWithOptions` directly. And seems bug comes from #2523 ? from the description of `PR`, I can't see any information about this added method, and @bpoplauschi may think `PR` only for exposure 😂 , we'd better list everything we changed in laters.

Reply:
These method are here to keep the compatible for UIKit's 5 methods. Previouslly we have only 4 because we don't use this simple `UIGraphicsBeginImageContext` because where we use always contains a scale args. Apple says:

> This function is equivalent to calling the UIGraphicsBeginImageContextWithOptions function with the opaque parameter set to NO and a scale factor of 1.0.

Reply:
Emm, I mean change 
```
void SDGraphicsBeginImageContext(CGSize size) {
#if SD_UIKIT || SD_WATCH
    UIGraphicsBeginImageContext(size);
#else
    SDGraphicsBeginImageContextWithOptions(size, NO, 1.0);
#endif
}
```
to 
```
void SDGraphicsBeginImageContext(CGSize size) {
    SDGraphicsBeginImageContextWithOptions(size, NO, 1.0);
}
```

Reply:
I guess since UIKit have the built-in implementation, we can just call them. Even if someday they change the documentation and don't follows what they says 😅

Reply:
Emm, you mean `Apple` may change the default value of `opaque` and `scale factor`? 🤔 Trust me, they would not, it would mess the `UI`. My suggestion is only for reference, my opinion is reduce redundant code and keep the way `Apple` does.

Reply:
Just follow what our documentation says, no extra assume.

> For UIKit, these methods just call the same method in `UIGraphics.h`. See the documentation for usage.